instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py

```diff
@@ -47,7 +47,7 @@
 from opentelemetry.instrumentation.dbapi.version import __version__
 from opentelemetry.instrumentation.utils import (
     _generate_opentelemetry_traceparent,
-    _generate_sql_comment,
+    generate_sql_comment,
```

Comment:
Why make these public?

Reply:
As we are importing the function across libraries I had changed it to private.

Reply:
Reverted the changes